src/DiffSharp.Core/DiffSharp.Compose.fs

```diff
@@ -8,7 +8,17 @@ module DiffSharp.Compose
 // static member sub(b:Tensor) = fun (a:Tensor) -> a.sub(b)
 
 type dsharp with
-    /// <summary>TBD</summary>
+    /// <summary>Returns a tensor where each row contains <paramref name="numSamples"/> indices sampled from the multinomial probability distribution located in the corresponding row of tensor input.</summary>
```

Comment:
In general I haven't been using `paramref` , partly because we don't generate any links for this.  There is checking for this in the latest F# tools to double check the reference at least, but tbh it's so clumsy to type it in I think we can use just `<c>... </c>` or plan text for now and take a separate pass to add paramref later

Reply:
Note 

* F# tools don't yet have documentation-aware rename-refactor for paramred and other references.  This makes cross-references of any kind a bit hard.  

* Only paramref are checked, other cross-references are not checked.